conf/app.conf

```diff
@@ -3,6 +3,7 @@ httpport = 8000
 runmode = dev
 SessionOn = true
 copyrequestbody = true
+db = "mysql"

```

Comment:
no quotes

Reply:
Used here: https://github.com/casbin/casdoor/pull/50/commits/4dc5b21c02b6ff7f98657f5cbfc9f6bd510e9b15#diff-009ed459acbafa374d33f30ee04034a1bf95db713f5abf5a0a45cfe9545c343fR30 
and here: https://github.com/casbin/casdoor/pull/50/commits/4dc5b21c02b6ff7f98657f5cbfc9f6bd510e9b15#diff-6e87f35c571ba4718e96ba663dec842335060094fe60f9066b2aaa04570f2c1dR39

Reply:
`db = "mysql"` -> `db = mysql`

Reply:
" is quote